src/components/JobsPanel/jobsPanel.util.js

```diff
@@ -334,14 +334,17 @@ export const generateTableDataFromDefaultData = (
   setDefaultDataIsLoaded
 ) => {
   const parameters = generateDefaultParameters(
-    Object.entries(defaultData.task.spec.parameters)
+    Object.entries(defaultData.task.spec.parameters ?? [])
```

Comment:
`Object.entries` expects an object. It _returns_ an array. I think we should do this:
```suggestion
    Object.entries(defaultData.task.spec.parameters ?? {})
```